sphinxcontrib/needs/builder.py

```diff
@@ -86,3 +87,10 @@ def build_needs_json(app, exception):
     needs_builder = NeedsBuilder(app)
     needs_builder.set_environment(app.env)
     needs_builder.finish()
+    
+    
+def delete_temp_dir(app, exception):
+
+    tempdir = os.path.join(app.srcdir, "_temp_needs")
```

Comment:
can we put this magic path to ``defaults.py``? E.g. ``TEMP_FoLder``
So other functions can reuse it and we have one single place to change it if needed later.